examples/cluster/plot_adjusted_for_chance_measures.py

```diff
@@ -117,7 +117,7 @@ def fixed_classes_uniform_labelings_scores(
 plots = []
 names = []
 
-style.use("seaborn-colorblind")
+style.use("seaborn-v0_8-colorblind")
```

Comment:
This change will be addressed at https://github.com/scikit-learn/scikit-learn/pull/24785
Please revert it.